packages/orbit-components/src/Select/types.d.ts

```diff
@@ -35,4 +35,5 @@ export interface Props extends Common.Globals, Common.SpaceAfter, Common.DataAtt
   readonly insideInputGroup?: boolean;
   readonly inlineLabel?: boolean;
   readonly customValueText?: Common.Translation;
+  readonly ariaLabel?: Common.Translation;
```

Comment:
This should be of type string

Reply:
yes, I had it there before, the AI bot confused me. 🙈